test/promql/tests/test_controller_reconciliation_errors.yaml

```diff
@@ -0,0 +1,133 @@
+evaluation_interval: 1m
+
+rule_files:
+  - ../extracted-rules.yaml
+
+tests:
+
+  # Alerted test cases
+  - interval: 1m
+    input_series:
+      # Controller with continuous errors greater than 1% so this is alerted
+      - series: 'controller_runtime_reconcile_errors_total{controller="c1-controller", namespace="prod-ns1", pod="pod-1", container="c1", service="s1"}'
+        values: '1+1x2'
```

Comment:
Now that we only need a few samples, I think the shorthand notation used throughout these unit test series makes them less understandable than explicitly putting the values. I have to think far less if written like this:
```yml
values: '1 2 3'
```